docs/features.md

```diff
@@ -132,6 +132,33 @@ $ gopass edit golang.org/gopher
 
 The `edit` command uses the `$EDITOR` environment variable to start your preferred editor where you can easily edit multi-line content. `vim` will be the default if `$EDITOR` is not set.
 
+### Adding OTP Secrets
+
+*Note: Depending on your security needs, it may not behoove you to store your OTP secrets alongside your passwords! Look into [Multiple Stores](https://github.com/justwatchcom/gopass/blob/master/docs/features.md#multiple-stores) if you need things to be separate!*
+
+Typically sites will display a QR code containing a URL that starts with `oauth://`. This string contains information about generating your OTPs and can be directly added to your password file. For example:
+
+```
+gopass golang.org/gopher
+secret1234
+otpauth://totp/golang.org:gopher?secret=ABC123
+```
```

Comment:
@dominikschulz I don't get those instructions or this is really faulty.
With both secret/password and totp in one entry getting a password is a mess. It doesn't get copied automatically anymore.

Without totp works correctly, copies password to clipboard
```term
$ gopass show golang.org/gopher
No safe content to display, you can force display with show -f.
Copying password instead.
✔ Copied golang.org/gopher to clipboard. Will clear in 45 seconds.
```
```
$ gopass show -f golang.org/gopher
secret1234
```
```
$ gopass totp golang.org/gopher

Error: No OTP entry found for golang.org/gopher: illegal base32 data at input byte 0
```
Edit and add totp line.
```
$ gopass edit golang.org/gopher
```
totp works nicely
```
gopass totp golang.org/gopher
493858 lasts 27s 	|---===========================|
```
show -f shows everything, ok fine
```
$ gopass show -f golang.org/gopher
secret1234
otpauth://totp/golang.org:gopher?secret=ABC123
```
**Doesn't copy anymore to clippboard**, but properly doesn't show password
```
$ gopass golang.org/gopher
otpauth://totp/golang.org:gopher?secret=ABC123
```
I need to use flag again to copy:
```
$ gopass show -c golang.org/gopher
✔ Copied golang.org/gopher to clipboard. Will clear in 45 seconds.
```

Reply:
That looks confusing indeed, however it's also confusing to discuss this in an old PR. Could you copy your comment to a new issue referencing this PR, so we can properly discuss it and see how we can improve the UX?